test/e2e/e2e_test.go

```diff
@@ -510,6 +510,118 @@ var _ = Describe("falcon", Ordered, func() {
 		})
 	})
 
+	Context("Falcon Sidecar Sensor with Falcon Secret", func() {
```

Comment:
This isn't going to have access to the secrets across namespaces on a fresh cluster for e2e. We could try using the controller-manager permissions to get it working:
- config/rbac/role.yaml
- config/rbac/role_binding.yaml
- config/rbac/service_account.yaml